src/binary-reader-logging.cc

```diff
@@ -638,6 +638,22 @@ Result BinaryReaderLogging::OnComdatEntry(ComdatType kind, Index index) {
   return reader_->OnComdatEntry(kind, index);
 }
 
+Result BinaryReaderLogging::BeginCodeMetadataSection(std::string_view name,
+                                                       Offset size) {
+  LOGF("BeginCodeMetadataSection('" PRIstringview "', size:%" PRIzd ")\n",
+       WABT_PRINTF_STRING_VIEW_ARG(name), size);
+  Indent();
+  return reader_->BeginCodeMetadataSection(name, size);
+}
+Result BinaryReaderLogging::OnCodeMetadata(Offset code_offset,
+                                             const void* data,
+                                             Address size) {
```

Comment:
indentation looks wrong here.   Run `git clang-format origin/main` to fix this stuff?

Reply:
I think I forgot to run clang-format after the last rebase. I will do it now.